sopel/plugins/callables.py

```diff
@@ -0,0 +1,586 @@
+        handler.allow_bots = getattr(obj, 'allow_bots', handler.allow_bots)
+        handler.allow_echo = getattr(obj, 'echo', handler.allow_echo)
+        handler.priority = getattr(obj, 'priority', handler.priority)
+        handler.output_prefix = getattr(
+            obj, 'output_prefix', handler.output_prefix)
+        handler._docs = getattr(obj, '_docs', handler._docs)
+
+        # rules
+        handler.events = getattr(obj, 'event', handler.events)
+        handler.ctcp = getattr(obj, 'ctcp', handler.ctcp)
+        handler.rules = getattr(obj, 'rule', handler.rules)
+        handler.rule_lazy_loaders = getattr(
+            obj, 'rule_lazy_loaders', handler.rule_lazy_loaders)
+        handler.find_rules = getattr(obj, 'find_rules', handler.find_rules)
+        handler.search_rules = getattr(
+            obj, 'search_rules', handler.search_rules)
+
+        # urls
+        handler.url_regex = getattr(obj, 'url_regex', handler.url_regex)
+        handler.url_lazy_loaders = getattr(
+            obj, 'url_lazy_loaders', handler.url_lazy_loaders)
+
+        # named rules
+        handler.commands = getattr(
+            obj, 'commands', handler.commands)
+        handler.nickname_commands = getattr(
+            obj, 'nickname_commands', handler.nickname_commands)
+        handler.action_commands = getattr(
+            obj, 'action_commands', handler.action_commands)
+
+        # rate limiting
+        handler.user_rate = getattr(obj, 'user_rate', None)
+        handler.channel_rate = getattr(obj, 'channel_rate', None)
+        handler.global_rate = getattr(obj, 'global_rate', None)
+        handler.user_rate_message = getattr(obj, 'user_rate_message', None)
+        handler.channel_rate_message = getattr(
+            obj, 'channel_rate_message', None)
+        handler.global_rate_message = getattr(obj, 'global_rate_message', None)
+        handler.default_rate_message = getattr(
+            obj, 'default_rate_message', None)
+        handler.unblockable = getattr(obj, 'unblockable', handler.unblockable)
+
+        return handler
+
+    @property
+    def is_limitable(self) -> bool:
+        """Check if the callable is subject to rate limiting.
+
+        :return: ``True`` if it is subject to rate limiting
+
+        Limitable callables aren't necessarily triggerable directly, but they
+        all must pass through Sopel's rate-limiting machinery during
+        dispatching.
+        """
+        allowed = any([
+            self.rules,
+            self.rule_lazy_loaders,
+            self.find_rules,
+            self.find_rules_lazy_loaders,
+            self.search_rules,
+            self.search_rules_lazy_loaders,
+            self.event,
+            self.ctcp,
+            self.commands,
+            self.nickname_commands,
+            self.action_commands,
+            self.url_regex,
+            self.url_lazy_loaders,
+        ])
+
+        return allowed
+
+    @property
+    def is_triggerable(self) -> bool:
```

Comment:
I went into too much detail about this in an earlier file… Won't repeat myself.

But the gist is that I think this property needs a better name. URL callbacks seem "triggerable" too, in a way not meaningfully different from how `find_rules` work.